WalletWasabi.Fluent/Models/UI/ApplicationSettings.cs

```diff
@@ -192,10 +195,25 @@ public ApplicationSettings(string persistentConfigFilePath, PersistentConfig per
 		this.WhenAnyValue(x => x.DoUpdateOnClose)
 			.Do(x => Services.UpdateManager.DoUpdateOnClose = x)
 			.Subscribe();
+
+		RunOnSystemStartupCommand = ReactiveCommand.Create(async () =>
+		{
+			try
+			{
+				RunOnSystemStartup = RunOnSystemStartup;
```

Comment:
I don't understand this. Can you explain?

Reply:
Small mistakes when moving this code, it should be `settings.RunOnSystemStartup`